src/Http/Http.Extensions/test/ProblemDetailsDefaultWriterTest.cs

```diff
@@ -600,6 +600,8 @@ await writer.WriteAsync(new ProblemDetailsContext()
     [InlineData("application/*")]
     [InlineData("application/json")]
     [InlineData("application/problem+json")]
+    [InlineData("application/json; charset=utf-8")]
+    [InlineData("application/json; v=1.0")]
     public void CanWrite_ReturnsTrue_WhenJsonAccepted(string contentType)
```

Comment:
To cover our bases, can we add a check for `*/*` as well?

Reply:
Already do.